lib/resources/cran.rb

```diff
@@ -0,0 +1,73 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Richter
+
+# Usage:
+# describe cran('DBI') do
+#   it { should be_installed }
+# end
+#
+
+module Inspec::Resources
+  class CranPackage < Inspec.resource(1)
+    name 'cran'
+    desc 'Use the `cran` InSpec audit resource to test R modules that are installed from CRAN package repository.'
+    example "
+      describe cran('DBI') do
+        it { should be_installed }
+      end
+    "
+
+    def initialize(package_name, perl_lib_path = nil)
+      @package_name = package_name
+      @r_cmd = 'Rscript'
+
+      # this resource is not supported on Windows
+      skip_resource 'The `cran` resource is not supported on your OS yet.' if inspec.os.windows?
```

Comment:
Same as above - can we add a `return` here so we don't check for the command's existence on windows?